From 594da92d09a914a2d553ceca86a8731885383b4a Mon Sep 17 00:00:00 2001
From: "Baoli.Zhang" <baoli.zhang@intel.com>
Date: Tue, 11 Mar 2025 13:02:45 +0800
Subject: [PATCH 46/76] Intel/non-rt-config: Enable the support for trusted
 compute

This change is to fix the guest os failure in trusted compute
feature (ITEP-23573).

Signed-off-by: Baoli.Zhang <baoli.zhang@intel.com>
---
 Intel/config | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Intel/config b/Intel/config
index d7ecabe1e658b..de9bc05968446 100644
--- a/Intel/config
+++ b/Intel/config
@@ -599,7 +599,7 @@ CONFIG_ACPI_HED=y
 # CONFIG_ACPI_CUSTOM_METHOD is not set
 # CONFIG_ACPI_BGRT is not set
 # CONFIG_ACPI_REDUCED_HARDWARE_ONLY is not set
-CONFIG_ACPI_NFIT=m
+CONFIG_ACPI_NFIT=y
 # CONFIG_NFIT_SECURITY_DEBUG is not set
 CONFIG_ACPI_NUMA=y
 # CONFIG_ACPI_HMAT is not set
@@ -6921,7 +6921,7 @@ CONFIG_RAS_CEC=y
 # end of Android
 
 CONFIG_LIBNVDIMM=y
-CONFIG_BLK_DEV_PMEM=m
+CONFIG_BLK_DEV_PMEM=y
 CONFIG_ND_CLAIM=y
 CONFIG_ND_BTT=m
 CONFIG_BTT=y
-- 
2.43.0

